XenAdmin/Controls/XenSearch/QueryPanel.cs

```diff
@@ -213,12 +213,14 @@ private static GridItemBase GetDefaultColumn(string s)
 
             if (s == "disks")
             {
-                return NewGridHeaderItem(Messages.OVERVIEW_DISKS, SortOrder.Descending, 120, 120);
+                int colWidth = Int32.Parse(resources.GetString("DISKS_OVERVIEW_COL_WIDTH"));

```

Comment:
I recommend using
```
int colWidth;
Int32.TryParse(resources.GetString("DISKS_OVERVIEW_COL_WIDTH"), out colWidth);
```
If parsing fails the colWidth wil be zero and the result will be ugly, but at least it doesn't throw exceptions.